tests/test_site.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from flask import g
+from flask import session
+
+def test_home_page(client, app):
+    # test that viewing the page renders without template errors
+    assert client.get("/").status_code == 200
+
+def test_output_model(client, app):
+    # test that viewing the page renders without template errors
+    assert client.get("/output_model").status_code == 200
```

Comment:
Not able to run these tests until later tonight, but they look like they will work. Excellent job!

I do have one suggest change, which is: use the `@pytest.mark.parametrize` decorator instead of writing a new test for each page. You can probably google for some examples of how that works, but basically you want to loop through the list `['/', '/output_model']` using that decorator.

The idea here is that as we expand the website, adding a new test for each page is a matter of adding another page to that list, instead of writing a whole new function each time.

Reply:
Very cool, thx for the explanation. I got this to work and committed it to my tests branch, but not sure if the changes are reflected in this PR. Basically it is list of tuples: 

@pytest.mark.parametrize('page, result',
                         [
                             ("/", 200),
                             ("/output_model", 200)
                         ]
                         )
def test_page(client, app, page, result):
    assert client.get(page).status_code == result

Reply:
That works! Excellent job. I'll checkout your branch and test the code before the PR, but based on a spot check I think this gets a LGTM.